src/main.dart

```diff
@@ -1,8 +1,193 @@
-// TODO: Run task list app
-void main(List<String> args) {
-  print('Hello World');
+import 'dart:io';
+
+class Task {
+  String description;
+  bool isCompleted;
+
+  Task(this.description, {this.isCompleted = false});
+}
+
+void main() {
+  final List<Task> tasks = [];
+
+  while (true) {
+    print("Task List");
+    print("1. task number.");
+    print("2. Add Title.");
+    print("3. add description.");
```

Comment:
These 3 functions should be combined into one function called "Add Task" where you ask the user all the necessary questions then add the task to the user